packages/react-bindings/src/styles/convertCssTimeToNumber.ts

```diff
@@ -0,0 +1,20 @@
+import * as _ from 'lodash'
+
+const convertCssTimeToNumber = (time: string): number => {
+  if (_.isNil(time) || time === 'initial' || time === 'inherit') {
+    return 0
+  }
+
+  if (!_.endsWith(time, 'ms') && !_.endsWith(time, 's')) {
```

Comment:
probably better to read if conditions are positive.

```suggestion
  if (_.endsWith(time, 'ms') || _.endsWith(time, 's')) {
```

Reply:
But it should be  `if (!(_.endsWith(time, 'ms') || _.endsWith(time, 's'))) {` in that case

Reply:
yeah, sorry about that!